packages/dataviews/src/dataform-controls/toggle-group.tsx

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
This file is almost a verbatim copy of the existing [radio](https://github.com/Automattic/wp-calypso/blob/c7f8b7d914fef4ee4ae032624a925df4f466a10d/packages/dataviews/src/dataform-controls/radio.tsx) control, with some modifications.